sdk/communication/Azure.Communication.Email/src/autorest.md

```diff
@@ -4,7 +4,8 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    -  https://github.com/apattath/azure-rest-api-specs-apattath/blob/fd6518a294e6968a77bbad6ce412e736c803337f/specification/communication/data-plane/Email/preview/2021-10-01-preview/CommunicationServicesEmail.json
+    - https://github.com/Azure/azure-rest-api-specs/blob/2beb92d3d6c70d683183192bab3beac8bef09322/specification/communication/data-plane/Email/preview/2023-01-15-preview/CommunicationServicesEmail.json
 generation1-convenience-client: true
```

Comment:
@m-nash could you please confirm the autorest settings. 

Also, I see CodeGenClient and CodeGenSupressType getting added to the non-generated folder. Are these added by default when generation1-convenience-client:true is set?

Reply:
since these are in the non-generatted folders, they were added manually by me for our customization.

Reply:
@apattath thanks for letting me know. Please wait on @m-nash to weigh in on some of those attributes before you check-in. We have a certain set of attribute combinations that can cause trouble long-term so we are trying to make sure they are reviewed by our TL on that space.

Reply:
@pallavit Michael is reviewing this PR, so I'm resolving this conversation. Please feel free to reopen if you think it's not resolved. Thanks!